src/test/java/nextstep/subway/documentation/PathDocumentation.java

```diff
@@ -34,11 +36,15 @@ void path() {
                 1250
         );
 
-        when(pathService.findPath(1L, 2L, PathType.DISTANCE)).thenReturn(pathResponse);
+        when(pathService.findPath(anyLong(), anyLong(), PathType.DISTANCE, anyInt())).thenReturn(pathResponse);
```

Comment:
수정 후 제대로 문서가 진행되시나요? 🤔 
```suggestion
        when(pathService.findPath(anyLong(), anyLong(), any(), anyInt())).thenReturn(pathResponse);
```


ResponseFieldsSnippet과 RequestParametersSnippet 에 대해서도 작성해 주면 좋을 것 같아요 😄